lib/home/instance_table_model.dart

```diff
@@ -15,19 +16,9 @@ class InstanceTableModel extends ChangeNotifier {
 
   List<LxdInstanceId> get instances => _instances ?? <LxdInstanceId>[];
   List<LxdInstanceId>? _instances;
-  void _updateInstances(List<LxdInstanceId>? instances) {
-    if (const ListEquality().equals(_instances, instances)) return;
```

Comment:
This may potentially get expensive because it ends up comparing the names and projects of the whole instance list so I thought it was better to simplify things and unconditionally `notifyListeners()` at the end of `_sortAndFilter()` instead because otherwise, I'd have to call `notifyListeners()` after `_sortAndFilter()` anyway.